containers/proxy-helm/values.yaml

```diff
@@ -1,6 +1,15 @@
 
-repository: registry.opensuse.org/uyuni
-version: latest
+defaultRegistry: registry.opensuse.org/uyuni
+defaultVersion: latest
```

Comment:
Renaming those would break existing users (if there is any) config. If possible I would avoid renaming.